{"uuid": "2defa1ba-d149-4a2f-87c5-be11cb159110", "children": ["7e9a6abb-9dd1-42ac-b8c4-9d33e350be03", "83e1f8b5-1fda-4867-87ed-962ee71d7fab", "cd5af145-b752-403b-864d-d3889e334cff", "764886f2-4ec3-4133-9bbf-fba32b9383be", "f241020e-880d-4a99-8e26-62cb07748651"], "befores": [{"name": "login", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"D:\\t\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"D:\\t\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"D:\\t\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"D:\\t\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\t\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\t\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 929, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Administrator\\Desktop\\test\\系统设置\\系统管理设置\\选项表单管理\\test_ERP分类配置.py\", line 135, in login\n    raise e\n  File \"C:\\Users\\Administrator\\Desktop\\test\\系统设置\\系统管理设置\\选项表单管理\\test_ERP分类配置.py\", line 108, in login\n    captcha_data = captcha_img.get_attribute(\"src\").split(\",\")[1]\n                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^\n"}, "attachments": [{"name": "登录失败截图", "source": "99f55fa9-162c-435b-964e-3883a7a3aac5-attachment.png", "type": "image/png"}], "start": 1753514181655, "stop": 1753514184015}], "start": 1753514181655, "stop": 1753514184364}